### ISSUE TITLE
Screen Reader: Settings-Profile: Status button does not announce selected option or no option

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#74887]
## Action Performed
Prerequisite:
- The user is logged in

1. Using JAWS + Chrome, open the page https://new.expensify.com/settings/profile
2. Navigate using TAB to the status icon, located next to the user name, in the middle navigation panel (emoji icon or blank smile face)
3. Observe how the element is announced

## Expected Result
The status icon, located next to the user name and email, will announce either the selected emoji or that there has been no selection made, e.g.:
- 'state <emoji>'
- 'state, not selected>'
or similar

## Actual Result
When users navigate to the 'State' emoji button, it is not announced with any information. It only announces as 'State button' regardless of whether or not an emoji icon has been selected:
- 'State'

HTML:
<button aria-label="State" role="button" data-selection-scrapper-hidden-element="true" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-9ffhgg r-lrvibr" type="button" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx"><div class="css-g5y9jx r-1awozwy r-rd2f51 r-y47klf r-1pl0ysf r-eu3ka r-1777fci r-1aockid"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 20 20" width="20" height="20" hovered="false" pressed="false" fill="#A2A9A3"><path d="M10 19c5 0 9-4 9-9s-4-9-9-9-9 4-9 9 4 9 9 9M6.5 8C7.3 8 8 7.3 8 6.5S7.3 5 6.5 5 5 5.7 5 6.5 5.7 8 6.5 8m3.5 8c2.2 0 4-2.2 4-5H6c0 2.8 1.8 5 4 5m5-9.5c0 .8-.7 1.5-1.5 1.5S12 7.3 12 6.5 12.7 5 13.5 5s1.5.7 1.5 1.5" style="fill-rule: evenodd; clip-rule: evenodd;"></path></svg></div></div></button>

## Area issue was found
Settings–Profile

## Failed Checkpoint
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User Impact
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested Resolution
Update the aria-label to include both - the visible name and entered option, e.g.:
- 'state <emoji>'
- 'state, not selected>'
or similar
https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA14

## Environment(s)
Windows Windows 10 - Chrome

## Screenshots/Videos
<img width="1917" height="996" alt="Image" src="https://github.com/user-attachments/assets/a59d9173-be74-4c7e-85ea-a2c646f7a42f" />

https://github.com/user-attachments/assets/cb7f168c-4866-4746-8d71-d432eb171019

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6994169
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by mallenexpensify** [comment]
`Accessibility` issue, will be addressed soon.

**Comment by mallenexpensify** [comment]
Accessibility, unassigning, will be addressed in bulk once all issues are created.
